br/backup-and-restore-faq.md

```diff
@@ -0,0 +1,57 @@
+---
+title: BR 常见问题解答
+summary: BR FAQ & troubleshooting
+category: FAQ
+---
+ 
+# BR 常见问题解答
+ 
+本文是 BR 的 FAQ，记录了一些在 BR 中的常见问题和解答。
+ 
+我们无法穷尽所有问题，如果遇到未在其中且无法解决的问题，[AskTUG](http://asktug.com) 社区中的友善成员很乐意解答您的问题。
+ 
+## BR 备份时，对写入影响多大？
+ 
+使用 sysbench 的 oltp_read_only 场景全速备份到非服务盘，对集群的影响依照表结构的不同，对集群 QPS 的影响在 15%~25% 之间。
```

Comment:
oltp_read_only does not write.

https://github.com/akopytov/sysbench/blob/master/src/lua/oltp_common.lua

Reply:
We have read-only test result only... Let me rename this field... 😅